src/tests/ftest/deployment/io_sys_admin.yaml

```diff
@@ -56,8 +56,8 @@ container:
   properties: cksum:crc16,cksum_size:16384,srv_cksum:on
   control_method: daos
 container_acl:
-  new_user: daos_ci_tester_1@
-  new_group: daos_ci_test_grp_1@
+  new_user: daos_ci_tester_1
+  new_group: daos_ci_test_grp_1
```

Comment:
oh so those users do exist, but we are just adding an extra @ causing the issue?

Reply:
A secondary problem is that when `useradd` fails, the test silently skips over it :)

Reply:
> But the DAOS ACL entry DOES need the `@`

I thought that was fixed to make it optional..

Reply:
I didn't realize. I was just modeling after some other tests. I'll make a note / ticket to address that here and other places. Thanks!